src/utils.ts

```diff
@@ -43,16 +43,40 @@ export function isFileInDirectory(directory: string, filePath: string): boolean
 }
 
 export async function getRemappings(rootPath: string): Promise<string[]> {
+  // First try the remappings.txt file
   try {
-    const filePath = path.join(rootPath, 'remappings.txt');
-    const fileContent = await fs.readFile(filePath, 'utf8');
+    return await getRemappingsFromFile(path.join(rootPath, 'remappings.txt'));
+  } catch (e) {
+    // If the remappings file does not exist, try foundry.toml
+    try {
+      return await getRemappingsFromConfig(path.join(rootPath, 'foundry.toml'));
```

Comment:
are those only 2 places where remmapings can be contained? In case of hardhat, where they are placed?